### ISSUE TITLE
Notebook: I can see author on the notebooks list item

### ISSUE THREAD

**Issue by mnorriswalker** [influxdata/ui#1381]
We would like the ability to view a notebook author on the notebooks list page. We can use the same details based list UX that we are using in other list pages with in the UI. 

![Notebooks List Page (9).png](https://images.zenhubusercontent.com/6058abb75f3c06c1d25467ae/2968f3d8-fb1a-4a64-95d5-7fbcae69ab6b)

### Acceptance Criteria

- A user can see an author for every notebook in the notebooks list page

**Comment by jzahnd** [comment]
Let's do some research on this on how we should store the author information given the circumstance a user has been deleted.

**Comment by drdelambre** [comment]
we currently dont store this information, so this is blocked by the idpe notebooksd data migration task

**Comment by russorat** [comment]
@jzahnd could you connect the issue blocking this task?

**Comment by jzahnd** [comment]
I think we are unblocked on this now that we have finished our migration. Moving into ready.

**Comment by drdelambre** [comment]
@genehynson what's the feasibility of this from the jwt notebooksd is getting sent?

**Comment by genehynson** [comment]
yup I have this information in the token received by notebooksd:

```
  "user": {
    "id": "081231169534b000",
    "name": "mcfly@influxdata.com",
    "status": "active"
  },
```

**Comment by genehynson** [comment]
rough thoughts on what needs to be done to complete this:

1) migration to notebooks table that adds a column "createdBy" (and maybe "updatedBy"?) 
2) whenever a notebook is created, notebooksd will get the `token.user.name` property and store it in the `createdBy` column
3) [optional] whenever a notebook is updated, notebooksd will get the `token.user.name` property and store it in the `updatedBy` column

These columns will be automatically returned whenever the notebook is retrieved via the API so the UI can display them

**Comment by drdelambre** [comment]
notebooks can only be created once, so having that column on the notebook makes sense, but for the updatedBy column, wouldn't we want that in a separate table so that we can keep an audit log?

separately, given the dances we've been required to do lately for migrations, is this still a 2?

**Comment by genehynson** [comment]
perhaps `lastUpdatedBy` is a better column name. I wasn't imagining creating a whole audit table - that seems out of scope, probably? But I thought it might be useful to see who modified a notebook last. 

Up to @asalem1 if he wants to change it, but there are no hurdles to jump through with migrations anymore. And in fact, he's already opened a PR: https://github.com/influxdata/idpe/pull/11892

**Comment by asalem1** [comment]
updateBy seems a little out of scope for this issue. It might be something we want to revisit later when we start getting into audit logs and trying to see the history behind resource generation through notebooks. Gonna leave it at created and go from there
